### PR TITLE
ce: rb5: fix some typos

### PR DESCRIPTION
**Pull request by ndechesne**
The rendering of the page was not correct, there were a few typos left
in the markdown.

Signed-off-by: Nicolas Dechesne <nicolas.dechesne@linaro.org>

### CODE REVIEW

**Comment by ndechesne** [comment]
I ended up spending a bit more time on RB5 docs today.. and make a few patches..

File: consumer/dragonboard/qualcomm-robotics-rb5/installation/board-recovery.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-permalink: /documentation/consumer/dragonboard/dragonboard845c/installation/board-recovery.md.html
+title: Qualcomm RB5 Board Recovery
+permalink: /documentation/consumer/dragonboard/qualcomm-robotics-rb5/installation/board-recovery.md.html
 ---
 
-# DragonBoard 845c Board Recovery
+# Qualcomm RB5 Board Recovery
 
-This page outlines steps needed to recover your DragonBoard 845c board from a bricked software state. This instruction set is suggested to those who are experiences boot issues due to a corrution in the file system and/or other software components.
+This page outlines steps needed to recover your Qualcomm RB5 board from a bricked software state. This instruction set is suggested to those who are experiences boot issues due to a corrution in the file system and/or other software components.
```
Comment (sgermanserrano) [inline consumer/dragonboard/qualcomm-robotics-rb5/installation/board-recovery.md:8]: Typo
```suggestion
This page outlines steps needed to recover your Qualcomm RB5 board from a bricked software state. This instruction set is suggested to those who are experiencing boot issues due to a corruption in the file system and/or other software components.
```

**Review by sgermanserrano — CHANGES_REQUESTED** [review]
@ndechesne Can you check the couple of typos

File: consumer/dragonboard/qualcomm-robotics-rb5/installation/board-recovery.md (hunk 1 above)
Comment (ndechesne) [reply consumer/dragonboard/qualcomm-robotics-rb5/installation/board-recovery.md]: thanks for the review! I fixed them (and also in db845c which was initially copied here!)

**Review by sgermanserrano — APPROVED** [review]

**Comment by nooryom** [comment]
ارجوا من حضرتكم إجابة الطلب وشكرا

**Pull request merged**